### ISSUE TITLE
Widget that automatically shows last member introductions in 'Our stories'

### ISSUE THREAD

**Issue by micheleminno** [humanetech-community/community-hub#41]
We can develop something that gets updates via discourse API about last introductions posted on the forum and show then in the 'Our stories' section.

So there will be always the lastly joined members.

**Comment by aschrijver** [comment]
The easiest way to do this, is to have the entire Introduce Yourself topic be a comment thread of an Introductions page.

A special widget would need programming, but can be nice idea as well. The only issue is that in Introduce Yourself people also respond to an intro and have discussion / reactions that cannot be treated as 'new entries'.

**Comment by micheleminno** [comment]
Yes, it should imply strict moderation to redirect those extra discussions in the 'Introduce Yourself' topic.

**Comment by aschrijver** [comment]
If you talk about moderation there are 2 approaches:

1. Moderation from the forum
2. Moderation from the website

With 1) the easiest way would be to clean the Introduce Yourself thread from responses, so only introductions remain and hence the widget list is clean too.. The hard way would involve e.g. writing a 'theme component' for Discourse that adds a toolbar button 'Add to introductions' or something.

With 2) there can be just a list of URL's to introduction posts that are added to a collection. This is probably the easiest way, and also avoids having the widget show 'polluted' entries.

Both methods add a significant amount of moderation overhead. The risk is that the moderation lags behind on new introductions and - with 2) - it becomes hard to update (needs a careful lookup process).

**Comment by aschrijver** [comment]
See #52 

Having a People Directory would be a better approach. This is at most a nice-to-have and has complications with permission + moderation, while a directory adds more value.

Proposing to close this.
